Sources/BSON/BSONDocument.swift

```diff
@@ -0,0 +1,240 @@
+import Foundation
```

Comment:
can we maybe keep the current name of the file and you can move it after this separately? it's impossible to see the diff in this file because it's all being considered new code